Sources/SessionManager/SessionManager.swift

```diff
@@ -122,14 +118,14 @@ public class SessionManager {
     }
 
     public func invalidateSession() async throws -> Bool {
-        guard let sessionID = sessionID else {
-            throw SessionManagerError.sessionIDAbsent
+        guard let sessionId = sessionId else {
+            throw SessionManagerError.sessionIdAbsent
         }
         do {
```

Comment:
pretty sure this do..catch is pointless here and can be removed?

Reply:
This is needed since we are doing all calculations inside this do..catch for invalidateSession API input.

Reply:
What mean is:

```
do {
   ... 
   let something = try doSomething()
} catch let error {
  throw error;
}
```

Is equivalent to just this:
```
  let something = try doSomething()
```

Outcome is the same.

Reply:
Should be able to remove both of these in this function.